From 6586670171fe10d5817a7e5b84ffab7b677b4006 Mon Sep 17 00:00:00 2001
From: Aleix Pol <aleixpol@kde.org>
Date: Wed, 13 Apr 2022 17:25:52 +0200
Subject: [PATCH 2/3] Keep PlasmaComponents.Highlight on its former behaviour

Otherwise some applets like Kickoff Legacy don't get a highlight

BUG: 452555
(cherry picked from commit 734b956c27bd76df7fed6979beb27c2f30cbd3bf)
---
 src/declarativeimports/plasmacomponents/qml/Highlight.qml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/declarativeimports/plasmacomponents/qml/Highlight.qml b/src/declarativeimports/plasmacomponents/qml/Highlight.qml
index 672e57231..3672050b7 100644
--- a/src/declarativeimports/plasmacomponents/qml/Highlight.qml
+++ b/src/declarativeimports/plasmacomponents/qml/Highlight.qml
@@ -4,6 +4,7 @@
     SPDX-License-Identifier: LGPL-2.0-or-later
 */
 
+import QtQuick 2.15
 import org.kde.plasma.extras 2.0 as PlasmaExtras
 
 /**
@@ -15,4 +16,5 @@ PlasmaExtras.Highlight
     id: root
     // At some point we had the "hover" property which was replaced with "hovered"
     property alias hover: root.hovered
+    hovered: true
 }
-- 
2.35.1

